draft-ietf-httpbis-proxy-status.md

```diff
@@ -99,7 +99,7 @@ Parameters on each member convey additional information about that intermediary'
 
 When adding a value to the Proxy-Status field, intermediaries SHOULD preserve the existing members of the field, to allow debugging of the entire chain of intermediaries handling the request.
 
-Proxy-Status MAY be sent in HTTP trailers. For example, if an intermediary is streaming a response and the upstream connection suddenly terminates, Proxy-Status can be appended to the trailers of the outgoing message (since the headers have already been sent). As with all trailers, it might be silently discarded along the path to the user agent, so this SHOULD NOT be done unless it is not possible to send it in headers, and an intermediary MUST NOT send Proxy-Status as a trailer field unless it has also sent a corresponding Proxy-Status header field in the same message, so that the trailer value's ordering relative to other intermediaries is preserved.
+Proxy-Status MAY be sent in HTTP trailers. For example, if an intermediary is streaming a response and the upstream connection suddenly terminates, Proxy-Status can be appended only to the trailers of the outgoing message, since the headers have already been sent. As with all trailers, it might be silently discarded along the path to the user agent, so this SHOULD NOT be done unless it is not possible to send it in headers, and an intermediary MUST NOT send Proxy-Status as a trailer field unless it has also sent a corresponding Proxy-Status header field in the same message, so that the trailer value's ordering relative to other intermediaries is preserved.
```

Comment:
```suggestion
Proxy-Status MAY be sent in HTTP trailers. For example, if an intermediary is streaming a response and the upstream connection suddenly terminates, Proxy-Status can be appended to the trailers of the outgoing message, since the headers have already been sent. As with all trailers, it might be silently discarded along the path to the user agent, so this SHOULD NOT be done unless it is not possible to send it in headers, and an intermediary MUST NOT send Proxy-Status as a trailer field unless it has also sent a corresponding Proxy-Status header field in the same message, so that the trailer value's ordering relative to other intermediaries is preserved.
```
'only' seems inappropriate here, since we subsequently advise that it needs to be sent as a header to be included in trailers.

Reply:
I accepted this change, but how am I misreading it? The sentence says: 

> [...], Proxy-Status can be **only** appended to the trailers of the outgoing message, **since the headers have already been sent**.

so we're referring to the case where headers have been already sent, forwarding of the response body has been interrupted, and the only way that we can send Proxy-Status is in the trailers. How is it inappropriate?

I'm not a native speaker, but every time I read it without "only", it just felt wrong.

Reply:
Ah, never mind, you're correct; I misread. I'll merge this and reinstate it.